src/plugins/janus_videoroom.c

```diff
@@ -3003,6 +3007,7 @@ static janus_videoroom_subscriber_stream *janus_videoroom_subscriber_stream_add(
 	/* Allocate a new subscriber stream instance */
 	janus_videoroom_subscriber_stream *stream = g_malloc0(sizeof(janus_videoroom_subscriber_stream));
 	stream->mindex = g_list_length(subscriber->streams);
+	stream->opaqueid = g_strdup(opaqueid);
```

Comment:
This assumes `opaqueid` is always non-NULL, which is wrong. You should only duplicate the string if it exists.